tests/time_tracker_api/time_entries/time_entries_namespace_test.py

```diff
@@ -204,18 +214,34 @@ def test_get_time_entry_should_succeed_with_valid_id(
     'time_tracker_api.time_entries.time_entries_dao.TimeEntriesCosmosDBDao.handle_date_filter_args',
     Mock(),
 )
+@patch(
+    'time_tracker_api.time_entries.time_entries_repository.TimeEntryCosmosDBRepository.create_sql_date_range_filter',
+    Mock(),
+)
+@patch(
+    'commons.data_access_layer.cosmos_db.CosmosDBRepository.generate_params',
+    Mock(),
+)
 @patch('msal.ConfidentialClientApplication', Mock())
 @patch('utils.azure_users.AzureConnection.get_token', Mock())
 @patch('utils.azure_users.AzureConnection.is_test_user')
 @patch('utils.azure_users.AzureConnection.get_test_user_ids')
 @pytest.mark.parametrize(
     'current_user_is_tester, expected_user_ids',
     [
-        (True, ['id1', 'id2']),
-        (False, ['id2']),
+        (True, ['id1', 'id1']),
+        (False, ['id1', 'id1']),
```

Comment:
just a question why we have the `ids` with the same value

Reply:
Ok, the test was split in two tests because there is a need to verify when method fetches time entries for test user and for no test user.